app/xds_api/urls.py

```diff
@@ -25,4 +25,16 @@
      path('experiences/<str:exp_hash>/interest-lists',
           views.add_course_to_lists,
           name='add_course_to_lists'),
+     path('interest-lists/managed',
```

Comment:
Might be semantics, but is there a difference between "owned" and "managed" ?

Reply:
Nope, I was going back and forth between those 2, will make the change.